### PR TITLE
SQL lesson fixes

### PR DESCRIPTION
**Pull request by fmichonneau**
Several --mostly minor-- modifications to the lesson.

### CODE REVIEW

File: sql.md [hunk 1]
```diff
@@ -319,8 +331,12 @@ using a GROUP BY clause
...
 If we want to group by multiple fields, we give GROUP BY a comma separated list.
 
-***EXERCISE: Write queries that return: 1. How many individuals were counted in
-   each year. 2. Average weight of each species in each year***
+> ### Challenge
+>
+> Write queries that return:
+>
+> 1. How many individuals were counted in each year.
+> 2. Average weight of each species in each year*
```
Comment (ethanwhite) [inline sql.md]: 'year*' -> 'year.'

File: sql.md [hunk 2]
```diff
@@ -89,18 +75,37 @@ and search tab in the right hand section of the screen.
...
+This is a real dataset that has been used in over 100 publications. We've
+simplified it just a little bit for the workshop, but you can download the
+[full dataset](http://esapubs.org/archive/ecol/E090/118/) and work with it using
+exactly the same tools we'll learn about today.
+
+
 Import
 ------
 
+1. Download the three CSV file from the [Portal Database](http://figshare.com/articles/Portal_Project_Teaching_Database/1314459)
```
Comment (ethanwhite) [inline sql.md]: 'three CSV file' -> 'three CSV files'

File: sql.md [hunk 3]
```diff
@@ -89,18 +75,37 @@ and search tab in the right hand section of the screen.
...
 1. Start a New Database **Database -> New Database**
 2. Start the import **Database -> Import**
 3. Select the file to import
-4. Give the table a name (or use the default)
+4. Give the table a name that match the file name (surveys, species, plots), or use the default
```
Comment (ethanwhite) [inline sql.md]: 'match' -> 'matches'

**Comment by ethanwhite** [comment]
Looks great @fmichonneau! I noted a couple of typos in-line. The only other thing is that that there's one more case of and `Exercise` that should be converted to a properly formatted `Challenge`. It's a the end of the Import section:

"***EXERCISE: Import the plots and species tables***"

**Comment by fmichonneau** [comment]
Thanks @ethanwhite! If you don't see anything else, please push the green button.

**Comment by ethanwhite** [comment]
Thanks @fmichonneau!

**Pull request merged**